packages/help-extension/src/index.tsx

```diff
@@ -135,7 +135,7 @@ const about: JupyterFrontEndPlugin<void> = {
         );
         const copyright = (
           <span className="jp-About-copyright">
-            {trans.__('© 2015-2021 Project Jupyter Contributors')}
+            {trans.__('© 2015-2022 Project Jupyter Contributors')}
```

Comment:
Would it make sense to do that dynamically?

```suggestion
            {trans.__('© 2015-%1 Project Jupyter Contributors', new Date().getFullYear())}
```

Reply:
I have thought to that also :) I use it in various projects. I this case I was thinking a hard-coded value was safer to be protected against wrong date on user laptop and to have a more official stake. I am fine with your suggestion.

Reply:
> This is a manual bump based on the discussions in https://github.com/jupyterlab/jupyterlab/pull/10052

As mentioned in the top comment, there are reasons for not doing that, more info in this comment: https://github.com/jupyterlab/jupyterlab/pull/10052#pullrequestreview-630568160 (you might have to expand since it's been marked as resolved)

Reply:
Thanks for the reference @jtpio